src/responder/common/cache_req/plugins/cache_req_object_by_id.c

```diff
@@ -90,6 +90,11 @@ cache_req_object_by_id_lookup(TALLOC_CTX *mem_ctx,
                               struct sss_domain_info *domain,
                               struct ldb_result **_result)
 {
+    if ((domain->id_max != 0 && data->id > domain->id_max)
+        || (data->id < domain->id_min)) {
+        DEBUG(SSSDBG_OP_FAILURE, "id exceeded min/max boundaries\n");
+        return EOK;
```

Comment:
Same comments as in the group lookup. I wonder if it makes sense to add a common function to avoid duplicating the same lines?